ci/setup_env.sh

```diff
@@ -43,12 +47,30 @@ function check_signed_tag() {
   fi
 }
 
+function  check_versions_match () {
+  local versions_to_check=("$@")
+
+  if [ "${#versions_to_check[@]}" -eq 1 ]; then
+    echo "Warning: ${FUNCNAME[0]} requires more than one version to be able to compare with.  The build is not going to be released ..."
+    export IS_A_RELEASE="false" && return
+  fi
+
+  for (( i=0; i<((${#versions_to_check[@]}-1)); i++ )); do
+    [ "${versions_to_check[$i]}" != "${versions_to_check[(($i+1))]}" ] &&
+    { echo -e "Warning: one or more module(s) versions do NOT match. The build is not going to be released ... !!!\nThe versions are ${versions_to_check[*]}"; export IS_A_RELEASE="false" && break; }
```

Comment:
I just spot a bug in this function. it has to be `&& return` vs `break` because `break` will get out of the loop and continue to export IS_A_RELEASE='true` which we DO not want